src/isql/isql.epp

```diff
@@ -9619,3 +9713,249 @@ static const char* charset_to_string(unsigned charset)
 
 	return UNKNOWN;
 }
+
+
+/// class PerTableStats
+
+const unsigned char PerTableStats::m_items[] =
+{
+	isc_info_read_seq_count, isc_info_read_idx_count, 
+	isc_info_insert_count, isc_info_update_count, isc_info_delete_count, 
+	isc_info_backout_count, isc_info_purge_count, isc_info_expunge_count,
+	isc_info_end
+};
+
+const char* PerTableStats::m_itemNames[] =
+{
+	" Natural", " Index",
```

Comment:
It would be cleaner if you use the space in the printf usage instead of in each array member.

Reply:
It allows to easily manage alignment and indentation of header names and see it "right now", without compile and run.